Gemfile

```diff
@@ -31,7 +31,7 @@ group :development, :test do
   gem 'rspec-rails', '~> 3.0'
   gem 'database_cleaner', '~> 1.6.1'
   gem 'factory_girl', '~> 4.4'
```

Comment:
No biggie, but might want to use this as a time to switch to [factory_bot](https://github.com/thoughtbot/factory_bot_rails) - could well be a separate PR though if it needs a bunch of renaming vars

Reply:
Probably best done in a separate PR?